### ISSUE TITLE
icacls page - no RC way for dos command users? ;)

### ISSUE THREAD

**Issue by AlbertoP64** [MicrosoftDocs/windowsserverdocs#3303]
TL;DR: Missing DOS return code documentation for command. No traces in web, and rumours are of contrasting behaviours observed.

Hi, I need to call icacls from dos (evelated) prompt,
just to backup some ACL.

I looked forth around throughout all the net, but nothing found about any command return code documentation, or discussion useful reply.
Except, uhm, other desperate crying that no info were available about, all over the world.

Is possible that this information could be added to the page?
I need to know how to catch any error in command execution and filter consequently.

Somewhere in searches I've also read rumours, that _possibly_
the return code could be 0 for ok or a system error code if > 0,
 but elsewhere I've also read that some experienced a return code of 0 fixed
even in case of error.
So, if I should prepare to filter in dos the output messages
just to know what's happened, I will prefer just to know it from the sources :)

I know that command prompt is antiquate, but for some fast work usually
it gets faster than powershell and wmi studying.

Regards
AlbertoP64  

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0a990d09-b5b1-3aba-de57-0aec79c12784
* Version Independent ID: 36d90c17-ba95-4ed6-de9d-f379dcad6c4d
* Content: [icacls](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/icacls#feedback)
* Content Source: [WindowsServerDocs/administration/windows-commands/icacls.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/administration/windows-commands/icacls.md)
* Product: **windows-server-threshold**
* Technology: **manage-windows-commands**
* GitHub Login: @coreyp-at-msft
* Microsoft Alias: **coreyp**

**Comment by JeremyRand** [comment]
Empirically, I find that on Windows 7, success yields return code 0, while "The system cannot find the file specified" yields return code 2.  I have no idea whether this interface is guaranteed to be stable.

**Comment by AlbertoP64** [comment]
Up. 17 months and still no one adding two lines to the doc page :-D it' so for those old technology question, i suppose ^_^ 
Thanks @JeremyRand for the info, and tanks @PRMerger13 (maybe robot?) for adding those very Ms specific labels I won't be able to determine.

**Comment by IngridAtMicrosoft** [comment]
Thanks for your dedication to our documentation. If we have been unable to review your issue in a timely manner, we sincerely apologize for the delayed response. We have captured the informtion you have provided for review. We are closing this issue for now, but if you feel that it's still a concern, please respond and let us know. If you determine another possible update to our documentation, please don't hesitate to reach out again. #please-close
